VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
                         }
                     }
-
                     @Override
                     public void onException(Exception ex) {
                         super.onException(ex);
-                        onUserLoginFailure(ex, null, null);
```

Comment:
Yeah, I guess there won't be any exception in the normal course of things, but there might be an `IOException` or some other unexpected exception.